form.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+const form = document.getElementById("contact");
+const email = document.getElementById("email");
+
+function isEmail(email) {
+  return email.match(
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/);
+}
```

Comment:
Kindly add at least one validation checking if the email is written in all lowercase. If the email has any uppercase you should display a message showing telling the user and preventing the form from submitting 😉